### PR TITLE
Postgres crud

### PR DESCRIPTION
**Pull request by cmonez**
Crud Operations for postgres db @jeremylfchen

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -56,72 +57,37 @@ app.get('/itemInformation/:itemId', (req, res) => {
...
-        },
-        details: {
-          additionalDetails: data[0].additionalDetails,
-        },
-      };
-
+  postgres
+    .getDataForSpecifiedId(itemId)
+    .then((formattedData) => {
       res.send(formattedData);
```
Comment (jeremylfchen) [inline server.js]: is there a reason for not setting the status code here?

File: server.js [hunk 2]
```diff
@@ -56,72 +57,37 @@ app.get('/itemInformation/:itemId', (req, res) => {
...
-        .catch((err) => {
-          res.status(404).send('error in getTitlesAndBrands: ', err);
-        });
-    }
-  } else if (itemId < 100 || itemId > 199) {
-    console.log(itemId);
-    res.status(404).send('Invalid itemId');
-  } else {
-    db.getTitleAndBrand(itemId)
+    const itemIds = itemsInArray.split(',').map((id) => Number(id));
```
Comment (jeremylfchen) [inline server.js:60]: You convert `itemId` to int here and in line 75, but don't convert it in lines 112, 124.

File: database-postgres/queries.js [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+    descriptions.primary_brand,
+    descriptions.sku,
+    descriptions.days_to_ship,
+    attributes.material,
+    attributes.length,
+    attributes.width,
+    directions.directions,
+    details.additional_details,
+    colors.*, descriptions.description
+    from descriptions
```
Comment (jeremylfchen) [inline database-postgres/queries.js:136]: Does this impact the query if `from` is lower case?

File: database-postgres/queries.js [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+    INNER JOIN
+    attributes
+    on descriptions.description_id = attributes.description_id
```
Comment (jeremylfchen) [inline database-postgres/queries.js:139]: Same questions for `on`

**Review by jeremylfchen — APPROVED** [review]
Looks good, I like the way you are handling errors by sending descriptive messages back to the client. I definitely need to do that lol. Just have a few questions about status codes and converting item id to ints, and some syntactical questions.

File: server.js (hunk 1 above)
Comment (cmonez) [reply server.js]: No there isnt. I should add that.

File: database-postgres/queries.js (hunk 3 above)
Comment (cmonez) [reply database-postgres/queries.js:136]: No it doesn't. Using uppercase for SQL is just a convention, you can use lowercase but I should probably change FROM to uppercase just to stick to that convention good eye.

**Pull request merged**